packages/api/src/schema/schema.yaml

```diff
@@ -273,7 +273,6 @@ components:
       table: webhook_response
       required:
         - webhookId
-        - eventId
         - statusCode
```

Comment:
Turns out the `statusCode` field **is** already required. No need to have those checks in the code :)